wurst/lib/LocalAssets.wurst

```diff
@@ -2,6 +2,7 @@ package LocalAssets
 
 public class LocalIcons
     static constant bTNGreenSentryWard = "ReplaceableTextures\\CommandButtons\\BTNGreenSentryWard.blp"
+    static constant btnForgeKit        = "ReplaceableTextures\\CommandButtons\\BTNNewForge.blp"
```

Comment:
Did you add this to `imports/`? I don't see it. I also don't think it makes much sense to call it "new".

Reply:
Realized I was only looking at the diff for the most recent commit when I couldn't find this. I see it now. You should still rename it though.

Reply:
yeah i just named it that because the old armory model was named forge and at the time i didnt want to erase anything. ill change that now though